docs/components/modeler/forms/form-element-library/forms-element-library-image.md

```diff
@@ -1,15 +1,15 @@
 ---
 id: forms-element-library-image
 title: Image view
-description: A form element to display an image
+description: Learn about the library image view form element to display an image.
```

Comment:
The element is just called 'image view', the library refers to the documentation structure.

Reply:
I'll launch a PR to adjust 👍